core/client/src/main/java/alluxio/client/file/policy/LocalFirstAvoidEvictionPolicy.java

```diff
@@ -32,8 +33,10 @@
  * to store the block, for the values mCapacityBytes minus mUsedBytes is not the available bytes.
  */
 @ThreadSafe
-public final class LocalFirstAvoidEvictionPolicy implements FileWriteLocationPolicy {
+public final class LocalFirstAvoidEvictionPolicy
```

Comment:
Adding a TODO to remind us to move it to the right place when support for FileWriteLocationPolicy is removed is sufficient. Thanks.